### PR TITLE
Closes softlayer/sl-ember-components#986

### PR DESCRIPTION
**Pull request by SpikedKira**
[Review on Reviewable](https://reviewable.io/reviews/softlayer/sl-ember-components/1028)

### CODE REVIEW

**Comment by notmessenger** [comment]
Not going to do any more review of this PR until the scenario commented on in the *tests/unit/components/sl-pagination-test.js* file is resolved.


---

Reviewed 1 of 3 files at r1.
Review status: 1 of 3 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

<sup>**[tests/unit/components/sl-pagination-test.js, line 101 \[r1\]](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028#-K1n_EOvgk81sstFQ1GM)** ([raw file](https://github.com/softlayer/sl-ember-components/blob/df0e876b1ac178557819610f7a223e4081f0e490/tests/unit/components/sl-pagination-test.js#L101)):</sup>
Are you working off of the latest version of the EmberCli-1.13.8 branch?  This test case already exists - https://github.com/softlayer/sl-ember-components/blob/EmberCli-1.13.8/tests/unit/components/sl-pagination-test.js#L37

If anything though, may want to change the position of it with the test suite.


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028)

**Comment by notmessenger** [comment]
Go ahead and merge in the latest changes anyways so that the outstanding failing lint and tests in this branch will be resolved and ensure are not going to be re-introduced.


---

Review status: 1 of 3 files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028)

**Comment by notmessenger** [comment]
Reviewed 2 of 3 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028)

**Comment by SpikedKira** [comment]
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---

<sup>**[tests/unit/components/sl-pagination-test.js, line 101 \[r1\]](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028#-K1n_EOvgk81sstFQ1GM)** ([raw file](https://github.com/softlayer/sl-ember-components/blob/df0e876b1ac178557819610f7a223e4081f0e490/tests/unit/components/sl-pagination-test.js#L101)):</sup>
This was branched from latest EmberCli-1.13.8 an hour and a half before the PR was created.  I think it was just bad timing; I am merging it back in now.


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028)

File: tests/unit/components/sl-pagination-test.js [hunk 1]
```diff
@@ -223,3 +223,28 @@ test( 'changePageBy() does nothing when busy is true', function( assert ) {
         'changePage action was not sent'
     );
 });
+
+test( 'Dependent keys are correct', function( assert ) {
```
Comment (notmessenger) [inline tests/unit/components/sl-pagination-test.js]: This test case already exists in this test suite.  Remove the other one?

**Comment by notmessenger** [comment]
Review status: 2 of 3 files reviewed at latest revision, 2 unresolved discussions.

---

<sup>**[tests/unit/components/sl-pagination-test.js, line 227 \[r2\]](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028#-K1do691z7FTfEzAt4m3-r2-227)** ([raw file](https://github.com/softlayer/sl-ember-components/blob/8073e3eb6aaec8a9fce32bf2b0f3f40c2bf8d90b/tests/unit/components/sl-pagination-test.js#L227)):</sup>
This test case already exists in this test suite. Remove the other one?


---


Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028)

**Comment by notmessenger** [comment]
Reviewed 1 of 1 files at r3.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



Comments from the [review on Reviewable.io](https://reviewable.io:443/reviews/softlayer/sl-ember-components/1028)

**Pull request merged**